boards/adafruit/feather_esp32s2/adafruit_feather_tft_esp32s2.dts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2025 Philipp Steiner <philipp.steiner@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include "adafruit_feather_esp32s2_common.dtsi"
+#include "feather_connector_tft.dtsi"
+
+/ {
+	model = "Adafruit Feather ESP32-S2 TFT";
+	compatible = "espressif,esp32s2";
+
+	chosen {
+		zephyr,display = &st7789v_tft;
+	};
+
+	aliases {
+		backlight = &led1;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		status = "okay";
+
+		led1: led_1 {
+			label = "TFT Backlight";
+			gpios = <&gpio1 45 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	/*
+	 * On-board regulator powers I2C pull-ups and external devices connected
+	 * via the STEMMA QT connector.
+	 */
+	i2c_reg: i2c_reg {
+		label = "I2C Power";
+		compatible = "power-domain-gpio";
+		#power-domain-cells = <0>;
+		enable-gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+	};
+
+	mipi_dbi {
+		compatible = "zephyr,mipi-dbi-spi";
+		spi-dev = <&spi2>;
+		dc-gpios = <&gpio1 39 GPIO_ACTIVE_HIGH>;
```

Comment:
`gpio1 39` and `gpio1 40` sounds odd. Have you checked whether this is really working?
It should be:
`gpio1 39` -> `gpio1 7`
`gpio1 40` -> `gpio1 8`

Reply:
Review the whole code regarding `gpio1`entries to make sure it is correct.

Reply:
They are for sure correct, I have a `adafruit_feather@C/esp32s2`, `adafruit_feather_tft/esp32s2` and a `adafruit_feather_tft_reverse/esp32s2` for testing and the pin testing stuff was the most time intensive part.

As a reference this can also be seen in #75844 -> https://github.com/zephyrproject-rtos/zephyr/pull/75844/files#diff-ff6b718ac56f37ab2acb3fa472ffd470edc7774af473aeaf5520311e04034301